src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -94,17 +99,87 @@ export class BucketDetailComponent implements OnInit {
         url: ["/" + this.bucketId],
       });
       this.fromObjects = params.fromRoute === "fromObjects";
-      this.fromLifeCycle = params.fromRoute === "fromLifeCycle";
+      this.fromLifeCycle = params.fromRoute === "fromLifeCycle";    
       this.backetUrl = this.bucketId;
       this.getAlldir();
       this.allTypes = [];
       this.getTypes();
     }
     );
+    this.copySelectedDir = window.sessionStorage['searchIndex'] != "" ? JSON.parse(window.sessionStorage.getItem("searchIndex")) : [];
+  }
+  clickOperate(){
```

Comment:
This code is in #220 as well. Why is it replicated here?

Reply:
Yes, I have re-uploaded and deleted the duplicate parts